util/opentelemetry-util-http/test-requirements.txt

```diff
@@ -0,0 +1,12 @@
+asgiref==3.7.2
+Deprecated==1.2.14
+importlib-metadata==6.11.0
+iniconfig==2.0.0
+packaging==24.0
+pluggy==1.5.0
+py-cpuinfo==9.0.0
+pytest==7.4.4
+pytest-benchmark==4.0.0
+tomli==2.0.1
+typing_extensions==4.10.0
+-e opentelemetry-instrumentation
```

Comment:
I'm not seeing usage of `opentelemetry-instrumentation` libraries anywhere in the tests? Is this a typo?

Reply:
Actually is a dependency of the package itself:
If I remove that I get:
```log
.tox/py312-test-util-http/lib/python3.12/site-packages/opentelemetry/util/http/httplib.py:30: in <module>
    from opentelemetry.instrumentation.instrumentor import BaseInstrumentor
E   ModuleNotFoundError: No module named 'opentelemetry.instrumentation'
```